invenio_queues/config.py

```diff
@@ -10,4 +10,12 @@
 
 from .utils import get_connection_pool
 
+QUEUES_BROKER_URL = None
+"""Provide a specific broker_url for queues.
+
+If the variable is not configured it falls back to the default BROKER_URL of
+our application.
+"""
```

Comment:
```suggestion
"""Broker URL for queues.

If the variable is not configured it falls back to the default ``BROKER_URL`` of
our application.
"""
```

Add also an explanation of what are the expected values for this, e.g. link to the Kombu connection string docs.

Reply:
```python
# in config.py
QUEUES_DEFINITIONS = []
"""Static queue definitions.
QUEUES_DEFINITIONS = [
    {'name': 'notifications', 'exchange': Exchange(...)},
]

# then in usage docs: 
app.config['QUEUES_DEFINITIONS'] = [...]
ext_queues = InvenioQueues(app)
ext_queues.queues
"""
```